templates/html/css/main.css

```diff
@@ -109,6 +109,10 @@ h1, h2, h3, h4, h5, h6 {
   z-index: 1;
 }
 
+.message-container > .p-8 {
```

Comment:
That's a hack. The class `p-8` actually means to set the padding of all sides the same size (2 rem). If you want to change padding go to where `message-container` class is applied and add the following classes: `py-0 px-8`.